Dockerfile

```diff
@@ -1,37 +1,54 @@
-FROM python:3.6-slim
+FROM python:3.6-slim@sha256:6f39e7dfc5158b351cfea004541fc85898452c2e4cbeb2b36f00c286fc957a88
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONPATH=/app/ \
+    PORT=8888
+
+EXPOSE $PORT
 
 # add a non-privileged user for installing and running
 # the application
 RUN groupadd -g 9000 kinto && \
```

Comment:
not sure why we use uid/guid 9000 here.  10001 is the right value in the Dockerflow spec.